packages/components/package.json

```diff
@@ -53,6 +53,7 @@
 		"re-resizable": "^6.0.0",
 		"react-dates": "^17.1.1",
 		"react-spring": "^8.0.20",
+		"react-use-gesture": "^7.0.9",
```

Comment:
Leveraged for improved gesture handling. Created by the folks behind `react-spring` ❤️- an animation library that we currently use within Gutenberg